app/src/main/java/org/apache/fineract/ui/online/customers/createcustomer/FormCustomerDetailsFragment.java

```diff
@@ -298,4 +313,32 @@ public void onAttach(Context context) {
                     + " must implement OnNavigationBarListener.CustomerDetails");
         }
     }
+
+    @OnClick(R.id.iv_edit_is_member)
+    void editIsMember() {
+        if (tvMemberStatus.getVisibility() == View.VISIBLE) {
+            tvMemberStatus.setVisibility(View.GONE);
+            cbIsmember.setVisibility(View.VISIBLE);
+            ivEditIsMember.setImageResource(R.drawable.ms_ic_check);
+        } else {
+            tvMemberStatus.setVisibility(View.VISIBLE);
+            cbIsmember.setVisibility(View.GONE);
+            ivEditIsMember.setImageResource(R.drawable.ic_edit_black_24dp);
+        }
+    }
+
+    @OnCheckedChanged(R.id.cb_is_member)
+    void setIsMemberStatus() {
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.LOLLIPOP) {
```

Comment:
All code looks good but we are supporting upto Android 4.1(API level 16) and this code wont work below Lollipop(API level 22). Wouldn't it lack change listener below Lollipop devices? Can't we have any alternative for lower levels `compat` version of check box. What do you say @EGOR-IND

Reply:
Yeah, I'm finding a good option for api level less than lollipop. I'll add that patch very soon into this PR😊.

Reply:
Ok 👍